packages/@yoda/flora/src/cli-native.cc

```diff
@@ -5,6 +5,7 @@
 #define ERROR_INVALID_URI -1
 #define ERROR_INVALID_PARAM -2
 #define ERROR_NOT_CONNECTED -3
+#define ERROR_JVM_API_FAILED -4
```

Comment:
```suggestion
#define ERROR_NAPI_FAILED -4
```

Use NAPI instead of JVM which is confused with Java VM.